Packs/CheckPointHEC/Integrations/CheckPointHEC/CheckPointHEC.yml

```diff
@@ -0,0 +1,209 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: CheckPointHEC
+  version: -1
+configuration:
+- section: Connect
+  display: Smart API URL (e.g. https://smart-api-dev-1-us.avanan-dev.net)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- section: Collect
+  display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- section: Collect
+  display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- section: Collect
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  defaultvalue: "10"
+  type: 0
+  required: false
+- section: Connect
+  display: ""
+  displaypassword: Client ID
+  name: client_id
+  type: 9
+  required: true
+  hiddenusername: true
+- display: ""
+  displaypassword: Client Secret
+  name: client_secret
+  type: 9
+  required: true
+  hiddenusername: true
+- section: Collect
+  advanced: true
+  display: Fetch alerts with status (ACTIVE, CLOSED)
+  name: alert_status
+  defaultvalue: ACTIVE
+  type: 15
+  required: false
+  options:
+  - ACTIVE
+  - CLOSED
+- section: Collect
+  advanced: true
+  display: Fetch alerts with type
+  name: alert_type
+  type: 0
+  required: false
+  additionalinfo: Comma-separated list of types of alerts to fetch. Types might change over time. Some examples are 'Bug' and 'Vulnerability'
+- section: Collect
+  display: Minimum severity of alerts to fetch
+  name: min_severity
```

Comment:
It seems that the min_severity param is not used in the script, please go through the yml file and delete unnecessary params (if there are).

Reply:
@mmhw The problem with those params is that validation does not pass if they are missing.

Reply:
Hi @chkp-christiang,
Can you please share the error, I can't reproduce it.